client/state/test/index.js

```diff
@@ -0,0 +1,47 @@
+require( 'lib/react-test-env-setup' )();
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import mockery from 'mockery';
+
+describe( 'state', () => {
+	var createReduxStore;
+	before( () => {
+		mockery.registerMock( 'lib/themes/middlewares.js', {
```

Comment:
Hmm I'm not sure what changed, but it looks like I don't need to mock this anymore.